components/advanced_text_editor/formatting_bar/formatting_bar.tsx

```diff
@@ -142,6 +142,12 @@ interface FormattingBarProps {
      * location of the advanced text editor in the UI (center channel / RHS)
      */
     location: string;
+
+    /*
+     * controls that enhance the message,
+     * e.g: message priority picker
+     */
+    enhanceControls?: React.ReactNodeArray;
```

Comment:
after thinking about that I would rather name this `additionalControls` instead ... `enhance` implies that the existing controls get better in some way